test/transform/classInheritance.js

```diff
@@ -0,0 +1,171 @@
+import createTestHelpers from '../createTestHelpers';
+const {expectTransform, expectNoChange} = createTestHelpers(['class']);
+
+export default function inheritanceTests() {
+  describe('Inheritance', () => {
+    describe('node.js require(\'util\').inherits', () => {
```

Comment:
I see a lot of escaped single quotes `\'` inside the test code - would be much simpler to just use double-quotes.